index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <script src="https://kit.fontawesome.com/1935d064dd.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap" rel="stylesheet">
+  <title>Summer Camp and Rock climbing Competition For Kids</title>
+</head>
+<body>
+  <nav class="Navbar">
+    <div class="nav-signin bg-dark d-flex hideOnMobile-f">
+      <ul class="nav-ul gap-close">
+        <li><a href="#" class="fs-medium"><i class="fa fa-facebook fc-white fs-medium"></a></i></li>
+        <li><a href="#" class="fs-medium"><i class="fa fa-twitter fc-white fs-medium"></a></i></li>
+        <li><a href="#" class="fc-white fs-small no-style">Logout</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">Register</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">MyPage</a></li>
+      </ul>
+    </div>
+    <div class="nav-items d-flex align-center">
+      <!-- <button type="button" class="menubars" id="Menu-button"> -->
+        <img class="menubars" src="./images/hamburger.png" alt="Menu">
+        <!-- <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span> -->
+      <!-- </button> -->
+      <div class="nav-brand">
+        <a href="#"><img src="./images/Summer-Camp.jpg" class="hideOnMobile-f logo" alt="The Summer Camp Logo"></img></a>
+      </div>
+      <div class="nav-links d-right" id="nav-options">
+        <ul class="nav-ul Mobile-Menu gap-wide" id="mobile-Menu">
+          <!-- <li class='d-right close-menu'><button type="button" class="hideOnDesktop-I no-style Dark-Blue" id="closemenu">&times;</button></li> -->
+          <i class="fas fa-times close crossicon"></i>
+          <li><a class="Dark-Blue fs-medium  no-style" href="About.html">About</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="#Program">Competition</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="#Sponsor">Sponsor</a></li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="Headline bg-light">
+    <p class="Headline-greetings">Hello! Adventure Lovers</p>   
+    <h1 class="h1-mobile Headline-title">Summer Camp and Mountain Climbing</h1>
+    <article>
+      A rock climbing competition is to be held 
+      on 26th july 2023 open for all ages. You can register your team with us. We have great prizes 
+      and Surprises both for the winners and participients. 
+    </article>
```

Comment:
- [ ] This is not a proper way of using the article tag, please read this [here](https://www.w3schools.com/tags/tag_article.asp) to have a clear and better understanding of its use, in the meanwhile, I would suggest you change it into a `p` tag